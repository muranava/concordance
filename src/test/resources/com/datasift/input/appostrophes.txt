We'll go fishing. She said 'I love fishing'.