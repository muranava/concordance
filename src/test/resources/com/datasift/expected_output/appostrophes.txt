fishing {2:0,1}
go {1:0}
i {1:1}
love {1:1}
said {1:1}
she {1:1}
we'll {1:0}